provider/cmd/run.go

```diff
@@ -418,7 +438,8 @@ func showErrorToUser(err error) error {
 	// If the error has a complete message associated with it then show it
 	clientResponseError, ok := err.(gateway.ClientResponseError)
 	if ok && 0 != len(clientResponseError.Message) {
-		fmt.Fprintf(os.Stderr, "provider error messsage:\n%v\n", clientResponseError.Message)
+		_, _ = fmt.Fprintf(os.Stderr, "provider error messsage:\n%v\n", clientResponseError.Message)
+		err = nil
```

Comment:
What does suppressing the error on this line do?

Reply:
In this case command will not print usage